src/Agent.Worker/Build/BuildCommandExtension.cs

```diff
@@ -16,6 +16,8 @@ public sealed class BuildCommandExtension : AgentService, IWorkerCommandExtensio
 
         public string CommandArea => "build";
 
+        public string SupportedHostType => string.Empty;
```

Comment:
In general all commands are expected to work in all host types. There are only few exceptions. 

IMO it is an overhead to keep the SupportedHostType in sync with the no. of host types given we have this property defined in multiple classes. I prefer using string.Empty to indicate it is supported in all hosts.